client/disputes/details/index.tsx

```diff
@@ -27,11 +27,19 @@ const DisputeDetails = ( {
 	query: { id: disputeId },
 }: {
 	query: { id: string };
-} ): JSX.Element => {
+} ): JSX.Element | null => {
 	const { dispute, isLoading, doAccept } = useDispute( disputeId );
 	const disputeObject = dispute || ( {} as Dispute );
 	const disputeIsAvailable = ! isLoading && dispute && disputeObject.id;
 
+	// Bail early and return nothing if the feature flag is not enabled.
+	// Here as a hint/reminder to delete this file when the feature flag is removed.
+	const isDisputeOnTransactionPageEnabled =
```

Comment:
Was going to say to mention this in https://github.com/Automattic/woocommerce-payments/issues/7289 but now I think it's not necessary because whoever will work on that issue will search for `isDisputeOnTransactionPageEnabled` and will find this.